cli/cmd/plugin/ui/web/tanzu-ui/node-server/src/routes/api/restApiEndpoints.js

```diff
@@ -23,70 +18,208 @@ const router = express.Router({
+    res.status(200);
+    res.json(appConfig.edition);
+});
+
 /**
- * Create an Azure resource group
+ * Mock route for connect Avi controller
  */
-router.post(`${ENDPOINT}/test`, (req, res) => {
-    winston.info('Mock UI POST API body param region');
-    if (req.body.region) {
-        res.status(201);
+router.post('/avi', (req, res) => {
+    winston.info('Mock TKG UI CONNECT AVI CONTROLLER API');
+    if ((req.body.host === 'avi.local') &&
+        (req.body.username === 'administrator' || req.body.username === 'admin') &&
+        (req.body.password === 'password')) {
+        res.status(200);
         res.json({});
     } else {
-        res.status(400);
-        res.json({ message: "Missing mock 'region'" });
+        res.status(403);
+        res.json({ message: 'incorrect username or password' });
     }
 });
 
-/*********************************   DOCKER   **********************************/
+/**
+ * Mock route for getting Avi clouds
+ */
+router.get('/avi/clouds', (req, res) => {
+    winston.info('Mock TKG UI FETCH AVI CLOUDS');
+    res.status(200);
+    res.json([
+        {
+            uuid: '631ad1fe-40c5-11eb-b378-0242ac130002',
+            name: 'Cloud-1',
+            location: 'west'
+        },
+        {
+            uuid: '77ab78e4-40c5-11eb-b378-0242ac130002',
+            name: 'Cloud-1-S',
+            location: 'south'
+        },
+        {
+            uuid: '7ca583c6-40c5-11eb-b378-0242ac130002',
+            name: 'Cloud-2',
+            location: 'east'
+        },
+        {
+            uuid: '8047129c-40c5-11eb-b378-0242ac130002',
+            name: 'Cloud-3',
+            location: 'emea'
+        },
+    ]);
+});
 
-router.get(`${ENDPOINT}/providers/docker/daemon`, (req, res) => {
-    winston.info('Mock TCE UI VALIDATE DOCKER DAEMON');
-    mockDockerDaemonCounter++;
+/**
+ * Mock route for getting Avi service engine groups
+ */
+router.get('/avi/serviceenginegroups', (req, res) => {
+    winston.info('Mock TKG UI FETCH AVI SERVICE ENGINE GROUPS');
     res.status(200);
-    res.json(
+    res.json([
         {
-            status: mockDockerDaemonCounter > 1 ? true : false
-        }
-    );
+            uuid: 'b4454eba-40c5-11eb-b378-0242ac130002',
+            name: 'Group-1',
+            location: 'cloud-77ab78e4-40c5-11eb-b378-0242ac130002'
+        },
+        {
+            uuid: 'b9f23350-40c5-11eb-b378-0242ac130002',
+            name: 'Group-1',
+            location: 'cloud-77ab78e4-40c5-11eb-b378-0242ac130002'
+        },
+        {
+            uuid: 'be23eb08-40c5-11eb-b378-0242ac130002',
+            name: 'Group-2',
+            location: 'cloud-7ca583c6-40c5-11eb-b378-0242ac130002'
+        },
+        {
+            uuid: 'c2b2a20e-40c5-11eb-b378-0242ac130002',
+            name: 'Group-3',
+            location: 'cloud-8047129c-40c5-11eb-b378-0242ac130002'
+        },
+    ]);
 });
 
 /**
- * Mock route for create docker cluster
+ * Mock route for getting Avi service engine groups
```

Comment:
nitpick: should be VIP networks in comments

Reply:
fixed